src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/ElasticSearchAnalysisGraph.java

```diff
@@ -200,7 +199,7 @@ public void construct() {
     HeapHealthDecider heapHealthDecider = new HeapHealthDecider(RCA_PERIOD, highHeapUsageClusterRca,
         largeHeapClusterRca);
     heapHealthDecider.addTag(TAG_LOCUS, LOCUS_MASTER_NODE);
-    heapHealthDecider.addAllUpstreams(Collections.singletonList(highHeapUsageClusterRca));
+    heapHealthDecider.addAllUpstreams(Arrays.asList(highHeapUsageClusterRca, largeHeapClusterRca));
```

Comment:
Did we add decider integ tests? How did they pass without this?

Reply:
No, we don't have decider integ tests yet.

> How did they pass without this?

Not having it in the upstream list won't necessarily fail the integration test even if we had one. The scheduler can at worst get the scheduling wrong and have the decider run before the RCA(because they'll be executed in parallel based on the distance from the leaf nodes). This causes race conditions which may or may not fail in the integ test. This upstream dependency was present in the 128G PR: #439 though, so this wouldn't have been caught then. Maybe at/after merge?